DuggaSys/microservices/showDuggaService/updateActiveUsers_ms.php

```diff
@@ -27,4 +29,33 @@
     $query->execute();
 }
 
-echo json_encode($active);
+echo json_encode(retrieveShowDuggaService(
```

Comment:
Declare the variables that are used as parameters here

Reply:
Added the variable assignments from the monorepo to ensure full coverage of all the variables.